docs/analyzer-rules/AZFW0006.md

```diff
@@ -0,0 +1,23 @@
+# AZFW0006: Symbol Not Found
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0006|
+| **Category** |[FunctionMetadataGeneration]|
```

Comment:
Not sure how useful this category value for users. The metadata generation is abstracted away from users. Many users don't even know we generate it behind the scene. We are mostly showing the diagnostic error in customer code (location).

Reply:
Ooo yeah I agree - any name suggestions? like "AzureFunctionsSyntax"?